docs/server/timing-events.md

```diff
@@ -33,6 +35,39 @@ The `setTimeout()` method sets a timer which executes a function or specified
 piece of code once after the timer expires. It returns [`Delayed`](#delayed)
 instance which identifies the interval, so you can manipulate it later.
 
+#### Example
+
+This MVP example shows a Room with: `setInterval()`, `setTimeout` and clearing a previously stored instance of type `Delayed`; along with showing the currentTime from the Room's clock instance.
+After 1 second 'Time now ' + `this.clock.currentTime` is `console.log`'d, and then after 10 seconds we clear the interval: `this.delayedInterval.clear();`.
+
+```typescript fct_label="TypeScript"
+import http from "http";
+import { Room, Client } from "colyseus";
+
+export class MyRoom extends Room {
+    public delayedInterval: any;
```

Comment:
This can be type of `Delayed`, imported from `import { Room, Client, Delayed } from "colyseus"; `

Reply:
Thanks, yes a good idea - have updated PR - wasn't totally sure on using that as a type without initializing it? - if you can think of a better way please let me know and i'll try it out and document it